src/raw/mod.rs

```diff
@@ -1049,7 +1050,10 @@ impl<T, A: Allocator + Clone> RawTable<T, A> {
         // space for.
         let min_size = usize::max(self.table.items, min_size);
         if min_size == 0 {
-            *self = Self::new_in(self.table.alloc.clone());
+            unsafe {
+                mem::replace(&mut self.table, RawTableInner::new())
+                    .drop_inner_table::<T, _>(&self.alloc, Self::TABLE_LAYOUT);
```

Comment:
For clarity, can you split this into 2 separate statements?
```rust
let old_table = mem::replace(...);
old_table.drop_inner_table(...);
```

Reply:
I splitted this and all other similar expressions